queries/javascript/textobjects.scm

```diff
@@ -17,7 +17,11 @@
   (function_declaration) @function.outer) @function.outer.start
 
 (arrow_function
-  body: (_) @function.inner) @function.outer
+  . body: (_) @function.inner) @function.outer
+
+(arrow_function
+  body: (statement_block . "{" . (_) @_start @_end (_)? @_end . "}"
```

Comment:
I think you can remove the first `@_end` `#make-range` can handle the case that `@_end` wasn't found.

Reply:
but it shouldn't do any harm

Reply:
Sorry I haven't had the time to respond. Thanks for merging my PR! Feel free to remove the redundant `@_end` in the future!